src/components/global-styles.tsx

```diff
@@ -153,6 +184,18 @@ const SizeStyles = css`
     padding: 5px 9px;
     z-index: 1;
   }
+
+  .margin-bottom-bigger {
+    margin-bottom: 24px !important;
+  }
+
+  .intro-card--item {
+    flex-basis: 0;
+  }
+
+  .home--action-buttons {
+    flex-direction: row-reverse;
+  }
```

Comment:
I see that these are cascading from the container type classes, but as this directly relates to a component, rather than a global style, we can add this to the component.

We could have a hook, `useViewType` etc, that's responsible for this, detecting the path to determine which it is, allowing the component to apply these conditional styles.